File: Domains/ApplicationDomain/Gym/Model/MyInBody/ObesityAnalysis.cs
using ApplicationDomain.Common;
using System;
using System.Collections.Generic;
using System.Text;

namespace ApplicationDomain.Gym.Model.MyInBody
{
    public class ObesityAnalysis
    {
        public TestedResult BMI { set; get; }
        public TestedResult PBF { set; get; }
        public Evaluation BMIEvaluation { set; get; }
        public Evaluation PBFEvaluation { set; get; }
    }
}

File: Applications/ClientApp/src/app/models/inbody/obesity-analysis.ts
import { TestedResult } from './tested-result';

export enum Evaluation {
  Normal,
  Under,
  SlightlyOver,
  Over,
}

export class ObesityAnalysis{
  bmiEvaluation: Evaluation;
  pbfEvaluation: Evaluation;
  bmi: TestedResult = new TestedResult();
  pbf: TestedResult = new TestedResult();
}
